src/datetime_queries/utils.rs

```diff
@@ -24,15 +24,11 @@ pub fn err(reason: &str) -> WasmError {
     WasmError::Guest(String::from(reason))
 }
 
-/// used to convert the last component of a path (in this cause, the hour of a day) into a string
+/// used to convert the last component of a path (in this case, the hour of a day) into a string
 pub fn get_last_component_string(path_tag: LinkTag) -> ExternResult<String> {
-    let hour_path = Path::try_from(&path_tag)?;
-    let hour_components: Vec<hdk::hash_path::path::Component> = hour_path.into();
-
-    let hour_bytes: &hdk::hash_path::path::Component =
-        hour_components.last().ok_or(err("Invalid path"))?;
-    let hour_str: String = hour_bytes.try_into()?;
-
+    let component_bytes = &path_tag.0[1..];
```

Comment:
I'm unfamiliar with this bit of syntax, can you say what it does?

Reply:
I just copied it over from the source code of path https://github.com/holochain/holochain/blob/15df3817736a2e5524abdfd2d774be267b13333e/crates/hdk/src/hash_path/path.rs#L369 since there was a bit of a change to paths and components and to properly mock the hdk calls that involved links I followed how holochain converted between strings, paths, components and vec<u8>